digiwf-integrations/digiwf-s3-integration/digiwf-s3-integration-client-example/pom.xml

```
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>example-s3-integration-client</artifactId>
+    <name>example-s3-integration-client</name>
+
+    <properties>
+        <logstash.encoder>7.0.1</logstash.encoder>
+    </properties>
+
+    <parent>
+        <groupId>io.muenchendigital.digiwf</groupId>
+        <artifactId>digiwf-s3-integration</artifactId>
+        <version>0.1.1</version>
```

```suggestion
        <version>0.12.0-SNAPSHOT</version>
```

We version all libraries, application, integrations, ... together. Therefore, we use the same version in every pom

---

ok. why do we dont't use project.version instead?